Assets/Scripts/Player/PlayerController.cs

```diff
@@ -9,6 +9,9 @@ public class PlayerController : MonoBehaviour {
     [SerializeField]
     private int currentHealth;
 
+    public int getMaxHealth() { return maxHealth; }

```

Comment:
You could write this method (and `getCurrentHealth`) more concisely: `public int getMaxHealth() => maxHealth;` ([documentation](https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/statements-expressions-operators/expression-bodied-members)).